projects/aws/bottlerocket-bootstrap/pkg/kubeadm/controlplane_init.go

```diff
@@ -125,7 +125,7 @@ func controlPlaneInit() error {
 	}
 
 	// finish kubeadm
-	cmd = exec.Command(kubeadmBinary, utils.LogVerbosity, "init", "--skip-phases", "preflight,kubelet-start,certs,kubeconfig,bootstrap-token,control-plane,etcd",
+	cmd = exec.Command(kubeadmBinary, utils.LogVerbosity, "init", "--skip-phases", "preflight,kubelet-start,certs,kubeconfig,bootstrap-token,control-plane,etcd,show-join-command",
```

Comment:
does this work in older versions of kube? i assume so since it just a list, id assume kubeadm is smart enough to skip over ones it doesnt know?